src/node-file-trace.ts

```diff
@@ -65,7 +66,8 @@ export class Job {
   public processed: Set<string>;
   public warnings: Set<Error>;
   public reasons: NodeFileTraceReasons = Object.create(null);
-  
+  resolve!: (id: string, parent: string, job: Job, cjsResolve: boolean) => string | string[];
```

Comment:
Can we be explicit if this is public/private?

Also why `!` here?

Reply:
fair 👍

Reply:
good call, made me revisit the implementation 👍